subsystems/mvdm/ntvdm/bios/vidbios.c

```diff
@@ -3178,6 +3467,36 @@ VOID WINAPI VidBiosVideoService(LPWORD Stack)
             break;
         }
 
+        /* Write Pixel At */
+        case 0x0C:
+        {
+            BYTE Page = getBH();
+
+            /* Validate page (0xFF = current video page) */
```

Comment:
If we believe Ralf Brown
http://www.ctyme.com/intr/rb-0104.htm
"BH is ignored if the current video mode supports only one page."

(which is a bit at odds with the other INT 10h functions)